_posts/2018-11-15-Rust-survey-2018.md

```diff
@@ -0,0 +1,168 @@
+![Which platform are you developing on](/images/2018-11-RustSurvey/15-Platform_developing_on.png)
+
+Linux continues to be a powerhouse among Rust developers, holding on to roughly 80% of Rust developers.  Windows usage has grown slightly from 31% last year to 34% this year, its second year in a row of growth.
+
+![Which platforms are you developing for](/images/2018-11-RustSurvey/16-Platforms_targeting.png)
+
+Linux and Windows continued to show strongly as targets for Rust applications. Other platforms held largely the same as last year, with one exception: WebAssembly. The new technology has showed impressive growth, nearly doubling from last year's 13% to this year's 24%.
+
+![What editors do you use](/images/2018-11-RustSurvey/17-Editors.png)
+
+Vim, the front-runner in editors for two years has now finally been bested by VSCode, which grew from 33.8% of Rust developers to 44.4% this year.
+
+## **Rust at work**
+
+![Do you use Rust at work](/images/2018-11-RustSurvey/18-Rust_at_work.png)
+
+Rust continues is slow-and-steady growth in the workplace. We're now seeing year-of-year growth of full-time and part-time Rust, growing from last years 4.4% full-time and 16.6% part-time to this year's **8.9% full-time** and **21.2% part-time**, a doubling of full-time Rust commercial use.  In total, Rust commercial use grew from 21% to just over 30% of Rust users.
+
+![Is your company evaluating Rust](/images/2018-11-RustSurvey/19-Company_evaluate_Rust.png)
+
+There is more room for Rust to grow into more companies, over a third of which users report aren't currently looking into evaluating Rust in the coming year.  When paired with the survey data that said that nearly half of non-users needed the company support, this shows the need for further company outreach or more company-focused information about Rust.
+
+## **Feeling welcome**
+
+![Do you feel welcome in the Rust community](/images/2018-11-RustSurvey/23-Do_you_feel_welcome.png)
+
+An important part of the Rust community efforts are ensuring that the Rust project is a welcoming place for its users. New users should feel encouraged to explore, share ideas, and generally be themselves.
+
+When asked, both current Rust users and non-users largely felt welcome, though over a quarter of responses weren't sure.  There was also some regional variation in these responses.  For example, responses on the Russian version of the survey showed double the percent of unwelcome feelings at 4%. Mainland China showed even more at 8%.
+
+There's a challenge here to help Rust communities worldwide feel like they are part of what makes Rust unique, as Rust continues to grow a strong presence in more areas of the world.
+
+![Are you underrepresented in tech](/images/2018-11-RustSurvey/24-Underrepresented.png)
+
+Underrepresented groups continue to improve slowly year-over-year. The survey also highlights some challenges, as the number of women is still lower than the industry average of women in programming fields.
```

Comment:
"Underrepresented groups continue to improve slowly year-over-year." sounds like the people in the groups are improving, like, their skills or something? Perhaps "The number of people in Rust who self-identify as being part of a group underrepresented in technology is growing slowly year-over-year."

Reply:
I mentioned this on discord, but comparing with the industry average to me reads like that is our _goal_, which it isn't -- our goal is to be better than the industry average. Can we reword this?

Reply:
@Manishearth - We do not have an explicit goal here, other than to be as welcoming as possible. The industry average gives us one way to measure our growth that we should at the very least be able to meet, if not surpass.